{{cookiecutter.example_name}}/Code.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright Insight Software Consortium
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import itk
+
+if len(sys.argv) != 3:
+    print("Usage: " + sys.argv[0] + "<InputFileName> "
+          "<OutputFileName>")
+    sys.exit(1)
+
+input_file_name = sys.argv[1]
+output_file_name = sys.argv[2]
+
+Dimension = 2
+PixelType = itk.UC
```

Comment:
Which way of specifying the pixel type should we privilege? This one or [this other one](https://github.com/InsightSoftwareConsortium/ITKExamples/blob/master/src/Filtering/ImageFeature/SegmentBloodVessels/Code.py)

Reply:
We can prefer support any pixel type with `itk.imread`. Or, if the downstream filters only support `float`, then we can read in as `float`.

Reply:
Removed the pixel type declaration in 5409bf6.